src/content/blog/2024_Microgrant_Impact.md

```diff
@@ -0,0 +1,57 @@
+--- 
+title: "Celebrating a Milestone: Over 100 Microgrants Awarded by Unitary Fund"
+author: Kallie Ferguson
+day: 11
+month: 6
+year: 2024
+--- 
+
+The Unitary Fund's microgrant program recently marked an incredible milestone: over 100 microgrants awarded! This achievement highlights the program's pivotal role in advancing open source quantum computing projects globally. Through strategic grants and open-source ecosystem development, Unitary Fund fosters a flourishing, multiplying effect within the broader quantum ecosystem. These small but impactful investments drive accessible and transparent progress, essential for unlocking the full potential of quantum computing.
+
+Thank you to our sponsors for making these microgrants possible! Core Members: [IBM Quantum](https://www.ibm.com/quantum), [DoraHacks](https://dorahacks.io/), [OQD](https://openquantumdesign.org/), and [Scientifica](https://www.scientifica.vc/); Supporting Members: [AWS](https://aws.amazon.com/braket/), [Microsoft](https://www.microsoft.com/), [Pasqal](https://www.pasqal.com/), [QC Ware](https://www.qcware.com/), [IonQ](https://ionq.com/) and [SandboxAQ](https://www.sandboxaq.com/). 
+
+To commemorate this event, we spoke with our current and former grant recipients to gauge the impact and reach of their supported projects. Here’s what we found.
+
+![Impact by the Numbers: 31 Countries Supported, 420+ Papers citing microgrant funded research, 40+ New Libraries, 400+ Contributors & Maintainers, 18.2k+ Commits (excluding QuTiP), 3 startups and 1 non-profit, $356k awarded in microgrants](/images/2024_microgrant_impact.png) 
+
+This impressive impact was obtained for less than the cost for a U.S. university to provide two fully funded PhD studentships from enrollment to graduation.  
```

Comment:
@KallieFerguson Can we add this to the summary graphic? E.g. maybe at the bottom below the amount say "< the cost of funding two US PhD students"

Reply:
@francespoblete Would you be able to make this update?